spec.bs

```diff
@@ -1050,7 +1054,8 @@ To <dfn>fill in a pending fenced frame config</dfn> given a [=fenced frame confi
 <div algorithm>
 
 To <dfn>asynchronously finish reporting</dfn> given a
-[=fencedframetype/fenced frame reporting map=] |reportingMap|, [=leading bid info=] |leadingBidInfo|,
+[=fencedframetype/fenced frame reporting map=] |reportingMap|, [=auction config=] |auctionConfig|,
+[=reporting context map=] |reportingContextMap|, [=leading bid info=] |leadingBidInfo|,

```

Comment:
Probably, yeah, since auctionReportInfo is largely computed from the fordebugOnly stuff in reportingContextMap + realtimeReportingContributions, and both the failed auction and async winner path do similar things to both.

https://pr-preview.s3.amazonaws.com/morlovich/turtledove/pull/1312.html#:~:text=Set%20auctionReportInfo%E2%80%99s%20real%20time%20reporting%20contributions%20map%20to%20realTimeContributionsMap.

I am not sure I can cleanly split the computation + sending for P.Agg. stuff, though.

Reply:
yeah, not sure how possible it is. It's fine as it is, we can revisit and see if there's a better way of managing it in the future.